client/Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM fedora:25
 MAINTAINER Luiz Carvalho <lucarval@redhat.com>
 
+ARG UPDATES
 ARG UPDATES_TESTING
 
-RUN dnf update -y && dnf install -y dnf-plugins-core && \
-    if [ $UPDATES_TESTING ]; then dnf config-manager --set-enable updates-testing; fi && \
-    dnf -y update && \
+RUN if [ $UPDATES ]; then dnf update -y; fi && \
```

Comment:
Yep, I was thinking we should be building a base osbs-box image first and then base other on top of it. It would simplify a lot of things